moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -665,10 +645,10 @@ double ServoCalcs::velocityScalingFactorForSingularity(const Eigen::VectorXd& co
 
   // Calculate a small change in joints
   Eigen::VectorXd new_theta;
-  kinematic_state_->copyJointGroupPositions(joint_model_group_, new_theta);
+  current_state_->copyJointGroupPositions(joint_model_group_, new_theta);
   new_theta += pseudo_inverse * delta_x;
-  kinematic_state_->setJointGroupPositions(joint_model_group_, new_theta);
-  auto new_jacobian = kinematic_state_->getJacobian(joint_model_group_);
+  current_state_->setJointGroupPositions(joint_model_group_, new_theta);
+  auto new_jacobian = current_state_->getJacobian(joint_model_group_);
```

Comment:
Although this `auto` is not a new change, I think it would be good idea to make it explicit Eigen type just to avoid any "Eigen + auto type" pitfalls.

Reply:
done